src/api/api.js

```diff
@@ -1 +1,5 @@
-export const BASE_URL = 'https://mate-api.herokuapp.com';
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export const getData = url => fetch(`${BASE_URL}/${url}`)
```

Comment:
if you add options argument to this method you can reuse it later to add or remove items from the server

```suggestion
export const getData = url => fetch(`${BASE_URL}/${url}`, options)
```